tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -138,6 +166,9 @@ Delete VCH with non-cVM in same RP
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.destroy "%{TEST_RESOURCE}/Resources/%{VCH-NAME}"
```

Comment:
I would suggest calling all cleanup functions using `TEARDOWN` (either test or suite level based on need), so to make sure they are run in any conditions instead of manually we doing it which is error prone.